website/docs/design-patterns/component-overrides.md

```diff
@@ -0,0 +1,413 @@
+---
+title: Component Overrides Atmos Design Pattern
+sidebar_position: 14
+sidebar_label: Component Overrides
+description: Component Overrides Atmos Design Pattern
+---
+
+# Component Overrides
+
+The **Component Overrides** Design Pattern describes the mechanism of modifying and overriding the configuration and behavior of groups of Atmos
+components in the current scope.
+
+This is achieved by using the `overrides` section in Atmos stack manifests.
+
+<br/>
+
+:::tip
+Refer to [Component Overrides](/core-concepts/components/overrides) for more information on the `overrides` section
+:::
+
+## Applicability
+
+Use the **Component Overrides** pattern when:
+
+- You need to modify or override the configuration and behavior of groups of Atmos components
```

Comment:
```suggestion
- You need to modify or override the configuration and behavior of groups of Atmos components. It is especially useful to "override" settings when dealing with multiple levels of inheritance.
```

Reply:
It's not really clear to me what the "groups of Atmos components" are. Is it a collection of components working together in a stack configuration?